File: sqlparser/src/sql/csgen/ISqlQueryVisitor.cs
using System;
using System.Collections;
using System.IO;
using System.Threading;
using System.Net;
using System.Net.Sockets;
using System.Text;
using RSSBus.core.j2cs;



using RSSBus.core;
using RSSBus;
namespace CData.Sql {


public interface ISqlQueryVisitor {
  void Visit(SqlTable table) ;
  void Visit(SqlStatement stmt) ;
  void Visit(SqlColumn column) ;
  void Visit(SqlConditionNode criteria) ;
}
}

File: sqlparser/src/sql/ISqlQueryVisitor.java
//@
package cdata.sql;
//@
/*#
using RSSBus.core;
using RSSBus;
namespace CData.Sql {
#*/

import rssbus.RSBException;

/*#public#*/ /*@*/public/*@*/interface ISqlQueryVisitor {
  void visit(SqlTable table) throws RSBException;
  void visit(SqlStatement stmt) throws RSBException;
  void visit(SqlColumn column) throws RSBException;
  void visit(SqlConditionNode criteria) throws RSBException;
}
